### PR TITLE
doc(release): Prepare release notes for modules next 22.10

### PR DESCRIPTION
**Pull request by tuntoja**
Prepare release notes for Centreon Modules next 22.10

### CODE REVIEW

File: i18n/fr/docusaurus-plugin-content-docs/version-22.10/releases/centreon-commercial-extensions.md [hunk 1]
```diff
@@ -141,6 +198,12 @@ Release date: `October 26, 2022`
 
 ## Centreon Auto Discovery
 
+### 22.10.1
+
+Release date: `soon`
+
+- [UI] Fixed a cosmetic issue in Bam & Autodiscovery table header
```
Comment (nmagniezCentreon) [inline i18n/fr/docusaurus-plugin-content-docs/version-22.10/releases/centreon-commercial-extensions.md:205]: remove BAM?

File: i18n/fr/docusaurus-plugin-content-docs/version-22.10/releases/centreon-commercial-extensions.md [hunk 2]
```diff
@@ -17,6 +17,46 @@ Retrouvez plus de détails sur la version 22.10 dans notre [post de blog](https:
...
+- [UI] Fixed Colors for a "Text" type element is ignored by the viewer. 
+- [UI] Fixed an issue when tooltip don't display for "old perfdata" plugin in the graph.
+- [UI] Fixed an issue when updating a resource that did not display the new resource name.
+- [UI] Fixed an issue with Host and Service icons configuration that were not handled properly.
+- [UI] Fixed bug when custom URI is set, some old media in map are not displayed correctly.
+- [UI] Fixed bug who display a blank page when user click on host name inside a map.
+- [UI] Fixed issue when Dark Theme is not apply on tooltip in Geo Views / MAP ENGINE.
+- [UI] Fixed issues and improved the map display (visibility, background and breadcrumb color).
+- [UI] Fixed the background layout not fitting large map.
+
```
Comment (nmagniezCentreon) [inline i18n/fr/docusaurus-plugin-content-docs/version-22.10/releases/centreon-commercial-extensions.md:51]: manque 
- update draw.io : "Update draw.io library" (15827)
- Fixed an issue that caused the unit not matching the axis on graph widget. (12115)

**Review by nmagniezCentreon — CHANGES_REQUESTED** [review]

File: i18n/fr/docusaurus-plugin-content-docs/version-22.10/releases/centreon-commercial-extensions.md [hunk 3]
```diff
@@ -17,6 +17,46 @@ Retrouvez plus de détails sur la version 22.10 dans notre [post de blog](https:
...
+#### Enhancements
+
+- [API] Improve api response when server is starting up.
+- [API] Removed deprecated REST endpoints.
```
Comment (nmagniezCentreon) [inline i18n/fr/docusaurus-plugin-content-docs/version-22.10/releases/centreon-commercial-extensions.md:55]: A supprimer, ca va faire peur alors que c'est des trucs qui n'étaient pas visibles

**Review by nmagniezCentreon — APPROVED** [review]

File: i18n/fr/docusaurus-plugin-content-docs/version-22.10/releases/centreon-commercial-extensions.md [hunk 4]
```diff
@@ -17,6 +17,49 @@ Retrouvez plus de détails sur la version 22.10 dans notre [post de blog](https:
...
+Release date: `soon`
+
+> You need to [execute the **configure.sh** script](../graph-views/map-web-install.md#étape-6--exécuter-le-script-configuresh) in order to apply some fixes provided by this release.
+
+#### Bug fixes
+- [API] Fixed an issue with create element in view that could break default icon.
+- [CEIP] Added more details for MAP engine telemetry.
+- [Editor] Fixed an issue in shapes edition modal that caused scrollbar to be reset when browsing images.
+- [Editor] Fixed view edition with some links having corrupted bend points.
+- [Editor] Update draw.io library.
```
Comment (smau-centreon) [inline i18n/fr/docusaurus-plugin-content-docs/version-22.10/releases/centreon-commercial-extensions.md:31]: To rephrase

File: i18n/fr/docusaurus-plugin-content-docs/version-22.10/releases/centreon-commercial-extensions.md [hunk 5]
```diff
@@ -17,6 +17,49 @@ Retrouvez plus de détails sur la version 22.10 dans notre [post de blog](https:
...
 
+### 22.10.5
+
+Release date: `soon`
+
+> You need to [execute the **configure.sh** script](../graph-views/map-web-install.md#étape-6--exécuter-le-script-configuresh) in order to apply some fixes provided by this release.
+
+#### Bug fixes
+- [API] Fixed an issue with create element in view that could break default icon.
+- [CEIP] Added more details for MAP engine telemetry.
```
Comment (smau-centreon) [inline i18n/fr/docusaurus-plugin-content-docs/version-22.10/releases/centreon-commercial-extensions.md:28]: To keep as is?

File: i18n/fr/docusaurus-plugin-content-docs/version-22.10/releases/centreon-commercial-extensions.md [hunk 6]
```diff
@@ -17,6 +17,49 @@ Retrouvez plus de détails sur la version 22.10 dans notre [post de blog](https:
...
+- [Editor] Fixed an issue in shapes edition modal that caused scrollbar to be reset when browsing images.
+- [Editor] Fixed view edition with some links having corrupted bend points.
+- [Editor] Update draw.io library.
+- [Install] Fixed an issue preventing the installation of centreon-map-engine on the same machine as centreon-map-server.
```
Comment (smau-centreon) [inline i18n/fr/docusaurus-plugin-content-docs/version-22.10/releases/centreon-commercial-extensions.md:32]:
```suggestion
- [Install] Fixed an issue preventing the installation of **centreon-map-engine** on the same machine as **centreon-map-server**.
```

File: i18n/fr/docusaurus-plugin-content-docs/version-22.10/releases/centreon-commercial-extensions.md [hunk 7]
```diff
@@ -17,6 +17,49 @@ Retrouvez plus de détails sur la version 22.10 dans notre [post de blog](https:
...
+- [Monitoring] Fixed an issue where MAP was not supporting performance data returned  by Cisco ASA plugin.
```
Comment (smau-centreon) [inline i18n/fr/docusaurus-plugin-content-docs/version-22.10/releases/centreon-commercial-extensions.md:33]:
```suggestion
- [Monitoring] Fixed an issue where MAP was not supporting performance data returned by Cisco ASA plugin.
```

File: i18n/fr/docusaurus-plugin-content-docs/version-22.10/releases/centreon-commercial-extensions.md [hunk 8]
```diff
@@ -17,6 +17,49 @@ Retrouvez plus de détails sur la version 22.10 dans notre [post de blog](https:
...
+- [Server] Fix migration issue with images that could stop migration process.
```
Comment (smau-centreon) [inline i18n/fr/docusaurus-plugin-content-docs/version-22.10/releases/centreon-commercial-extensions.md:34]:
```suggestion
- [Server] Fixed a migration issue with images that could stop the migration process.
```

File: versioned_docs/version-22.10/releases/centreon-commercial-extensions.md [hunk 9]
```diff
@@ -18,12 +18,55 @@ Read more about version 22.10 in our [blog post](https://www.centreon.com/en/blo
...
 
 #### Bug fixes
+- [API] Fixed an issue with create element in view that could break default icon.
+- [CEIP] Added more details for MAP engine telemetry.
+- [Editor] Fixed an issue in shapes edition modal that caused scrollbar to be reset when browsing images.
+- [Editor] Fixed view edition with some links having corrupted bend points.
+- [Editor] Update draw.io library.
+- [Install] Fixed an issue preventing the installation of centreon-map-engine on the same machine as centreon-map-server.
+- [Monitoring] Fixed an issue where MAP was not supporting performance data returned  by Cisco ASA plugin.
+- [Server] Fix migration issue with images that could stop migration process.
```
Comment (smau-centreon) [inline versioned_docs/version-22.10/releases/centreon-commercial-extensions.md:37]:
```suggestion
- [Server] Fixed a migration issue with images that could stop the migration process.
```

File: versioned_docs/version-22.10/releases/centreon-commercial-extensions.md [hunk 10]
```diff
@@ -18,12 +18,55 @@ Read more about version 22.10 in our [blog post](https://www.centreon.com/en/blo
...
+Release date: `soon`
+
+> You need to [execute the **configure.sh** script](../graph-views/map-web-install.md#étape-6--exécuter-le-script-configuresh) in order to apply some fixes provided by this release.
 
 #### Bug fixes
+- [API] Fixed an issue with create element in view that could break default icon.
+- [CEIP] Added more details for MAP engine telemetry.
+- [Editor] Fixed an issue in shapes edition modal that caused scrollbar to be reset when browsing images.
+- [Editor] Fixed view edition with some links having corrupted bend points.
+- [Editor] Update draw.io library.
```
Comment (smau-centreon) [inline versioned_docs/version-22.10/releases/centreon-commercial-extensions.md:34]: To rephrase

File: i18n/fr/docusaurus-plugin-content-docs/version-22.10/releases/centreon-commercial-extensions.md [hunk 11]
```diff
@@ -17,6 +17,49 @@ Retrouvez plus de détails sur la version 22.10 dans notre [post de blog](https:
...
+- [Server] Fixed an issue that caused server to crash on corrupted metrics.
+- [Server] Fixed an issue that prevent the migration between MAP legacy and MAP.
+- [Server] Fixed an issue that prevented access to MAP in HTTPS.
+- [Server] Fixed an issue to allow more than 25 maps to be migrated.
+- [Server] Fixed an issue with authentication when several sessions are active.
+- [Server] Fixed an issue with insert an object in a link prevented the map from being saved.
+- [Server] Fixed an issue with map sharing.
+- [Server] Fixed configure.sh script to include storage event from broker output.
```
Comment (smau-centreon) [inline i18n/fr/docusaurus-plugin-content-docs/version-22.10/releases/centreon-commercial-extensions.md:42]:
```suggestion
- [Install] Fixed the **configure.sh** script to include storage event from Broker output.
```

File: i18n/fr/docusaurus-plugin-content-docs/version-22.10/releases/centreon-commercial-extensions.md [hunk 12]
```diff
@@ -17,6 +17,49 @@ Retrouvez plus de détails sur la version 22.10 dans notre [post de blog](https:
...
+- [Server] Fixed issue that made new hosts and services unavailable until restart.
```
Comment (smau-centreon) [inline i18n/fr/docusaurus-plugin-content-docs/version-22.10/releases/centreon-commercial-extensions.md:43]:
```suggestion
- [Server] Fixed an issue that made new hosts and services unavailable until restart.
```

File: versioned_docs/version-22.10/releases/centreon-commercial-extensions.md [hunk 13]
```diff
@@ -18,12 +18,55 @@ Read more about version 22.10 in our [blog post](https://www.centreon.com/en/blo
...
+- [Server] Fix migration issue with images that could stop migration process.
+- [Server] Fixed an issue that caused server to crash on corrupted metrics.
+- [Server] Fixed an issue that prevent the migration between MAP legacy and MAP.
+- [Server] Fixed an issue that prevented access to MAP in HTTPS.
+- [Server] Fixed an issue to allow more than 25 maps to be migrated.
+- [Server] Fixed an issue with authentication when several sessions are active.
+- [Server] Fixed an issue with insert an object in a link prevented the map from being saved.
+- [Server] Fixed an issue with map sharing.
+- [Server] Fixed configure.sh script to include storage event from broker output.
+- [Server] Fixed issue that made new hosts and services unavailable until restart.
```
Comment (smau-centreon) [inline versioned_docs/version-22.10/releases/centreon-commercial-extensions.md:46]:
```suggestion
- [Server] Fixed an issue that made new hosts and services unavailable until restart.
```

File: versioned_docs/version-22.10/releases/centreon-commercial-extensions.md [hunk 14]
```diff
@@ -18,12 +18,55 @@ Read more about version 22.10 in our [blog post](https://www.centreon.com/en/blo
...
+- [Server] Fixed the server startup error due to PF without virtual hosts.
```
Comment (smau-centreon) [inline versioned_docs/version-22.10/releases/centreon-commercial-extensions.md:47]:
```suggestion
- [Server] Fixed an issue with the server startup error caused by platform without virtual hosts.
```

File: versioned_docs/version-22.10/releases/centreon-commercial-extensions.md [hunk 15]
```diff
@@ -18,12 +18,55 @@ Read more about version 22.10 in our [blog post](https://www.centreon.com/en/blo
...
+- [UI] Fixed Colors for a "Text" type element is ignored by the viewer.
```
Comment (smau-centreon) [inline versioned_docs/version-22.10/releases/centreon-commercial-extensions.md:48]:
```suggestion
- [Viewer] Fixed an issue with colors for a "Text" type element that was ignored by the viewer.
```

File: i18n/fr/docusaurus-plugin-content-docs/version-22.10/releases/centreon-commercial-extensions.md [hunk 16]
```diff
@@ -17,6 +17,49 @@ Retrouvez plus de détails sur la version 22.10 dans notre [post de blog](https:
...
+- [Server] Fixed the server startup error due to PF without virtual hosts.
+- [UI] Fixed Colors for a "Text" type element is ignored by the viewer.
+- [UI] Fixed an issue when tooltip don't display for "old perfdata" plugin in the graph.
```
Comment (smau-centreon) [inline i18n/fr/docusaurus-plugin-content-docs/version-22.10/releases/centreon-commercial-extensions.md:46]:
```suggestion
- [UI] Fixed an issue where tooltips were not displayed for "old perfdata" plugin in the graph.
```

File: versioned_docs/version-22.10/releases/centreon-commercial-extensions.md [hunk 17]
```diff
@@ -18,12 +18,55 @@ Read more about version 22.10 in our [blog post](https://www.centreon.com/en/blo
...
+- [UI] Fixed an issue when tooltip don't display for "old perfdata" plugin in the graph.
```
Comment (smau-centreon) [inline versioned_docs/version-22.10/releases/centreon-commercial-extensions.md:49]:
```suggestion
- [UI] Fixed an issue where tooltips were not displayed for "old perfdata" plugin in the graph.
```

File: i18n/fr/docusaurus-plugin-content-docs/version-22.10/releases/centreon-commercial-extensions.md [hunk 18]
```diff
@@ -17,6 +17,49 @@ Retrouvez plus de détails sur la version 22.10 dans notre [post de blog](https:
...
+- [UI] Fixed an issue when updating a resource that did not display the new resource name.
```
Comment (smau-centreon) [inline i18n/fr/docusaurus-plugin-content-docs/version-22.10/releases/centreon-commercial-extensions.md:47]:
```suggestion
- [UI] Fixed an issue where updating a resource was not displaying the new resource name.
```

File: versioned_docs/version-22.10/releases/centreon-commercial-extensions.md [hunk 19]
```diff
@@ -18,12 +18,55 @@ Read more about version 22.10 in our [blog post](https://www.centreon.com/en/blo
...
+- [UI] Fixed an issue when updating a resource that did not display the new resource name.
```
Comment (smau-centreon) [inline versioned_docs/version-22.10/releases/centreon-commercial-extensions.md:50]:
```suggestion
- [UI] Fixed an issue where updating a resource was not displaying the new resource name.
```

File: i18n/fr/docusaurus-plugin-content-docs/version-22.10/releases/centreon-commercial-extensions.md [hunk 20]
```diff
@@ -17,6 +17,49 @@ Retrouvez plus de détails sur la version 22.10 dans notre [post de blog](https:
...
+- [UI] Fixed an issue with Host and Service icons configuration that were not handled properly.
+- [UI] Fixed bug when custom URI is set, some old media in map are not displayed correctly.
+- [UI] Fixed bug who display a blank page when user click on host name inside a map.
```
Comment (smau-centreon) [inline i18n/fr/docusaurus-plugin-content-docs/version-22.10/releases/centreon-commercial-extensions.md:50]:
```suggestion
- [UI] Fixed an issue where a blank page was displayed when the user clicked on a host name inside a map.
```

File: versioned_docs/version-22.10/releases/centreon-commercial-extensions.md [hunk 21]
```diff
@@ -18,12 +18,55 @@ Read more about version 22.10 in our [blog post](https://www.centreon.com/en/blo
...
+- [UI] Fixed an issue with Host and Service icons configuration that were not handled properly.
+- [UI] Fixed bug when custom URI is set, some old media in map are not displayed correctly.
+- [UI] Fixed bug who display a blank page when user click on host name inside a map.
```
Comment (smau-centreon) [inline versioned_docs/version-22.10/releases/centreon-commercial-extensions.md:53]:
```suggestion
- [UI] Fixed an issue where a blank page was displayed when the user clicked on a host name inside a map.
```

File: i18n/fr/docusaurus-plugin-content-docs/version-22.10/releases/centreon-commercial-extensions.md [hunk 22]
```diff
@@ -17,6 +17,49 @@ Retrouvez plus de détails sur la version 22.10 dans notre [post de blog](https:
...
+- [UI] Fixed issue when Dark Theme is not apply on tooltip in Geo Views / MAP ENGINE.
```
Comment (smau-centreon) [inline i18n/fr/docusaurus-plugin-content-docs/version-22.10/releases/centreon-commercial-extensions.md:51]:
```suggestion
- [UI] Fixed an issue where dark theme was not applied on tooltips in Geo views.
```

File: versioned_docs/version-22.10/releases/centreon-commercial-extensions.md [hunk 23]
```diff
@@ -18,12 +18,55 @@ Read more about version 22.10 in our [blog post](https://www.centreon.com/en/blo
...
+- [UI] Fixed issue when Dark Theme is not apply on tooltip in Geo Views / MAP ENGINE.
```
Comment (smau-centreon) [inline versioned_docs/version-22.10/releases/centreon-commercial-extensions.md:54]:
```suggestion
- [UI] Fixed an issue where dark theme was not applied on tooltips in Geo views.
```

File: i18n/fr/docusaurus-plugin-content-docs/version-22.10/releases/centreon-commercial-extensions.md [hunk 24]
```diff
@@ -17,6 +17,49 @@ Retrouvez plus de détails sur la version 22.10 dans notre [post de blog](https:
...
+- [UI] Fixed issues and improved the map display (visibility, background and breadcrumb color).
+- [UI] Fixed the background layout not fitting large map.
```
Comment (smau-centreon) [inline i18n/fr/docusaurus-plugin-content-docs/version-22.10/releases/centreon-commercial-extensions.md:53]:
```suggestion
- [UI] Fixed the background layout that was not fitting large maps.
```

File: versioned_docs/version-22.10/releases/centreon-commercial-extensions.md [hunk 25]
```diff
@@ -18,12 +18,55 @@ Read more about version 22.10 in our [blog post](https://www.centreon.com/en/blo
...
+- [UI] Fixed issues and improved the map display (visibility, background and breadcrumb color).
+- [UI] Fixed the background layout not fitting large map.
```
Comment (smau-centreon) [inline versioned_docs/version-22.10/releases/centreon-commercial-extensions.md:56]:
```suggestion
- [UI] Fixed the background layout that was not fitting large maps.
```

File: versioned_docs/version-22.10/releases/centreon-commercial-extensions.md [hunk 26]
```diff
@@ -18,12 +18,55 @@ Read more about version 22.10 in our [blog post](https://www.centreon.com/en/blo
...
+- [Viewer] Fixed an issue that caused the unit not matching the axis on graph widget.
+
+#### Enhancements
+
+- [API] Improve api response when server is starting up.
```
Comment (smau-centreon) [inline versioned_docs/version-22.10/releases/centreon-commercial-extensions.md:61]:
```suggestion
- [API] Improved the API response when server is starting up.
```

File: versioned_docs/version-22.10/releases/centreon-commercial-extensions.md [hunk 27]
```diff
@@ -18,12 +18,55 @@ Read more about version 22.10 in our [blog post](https://www.centreon.com/en/blo
...
+- [Editor] Manage font properties in editor.
```
Comment (smau-centreon) [inline versioned_docs/version-22.10/releases/centreon-commercial-extensions.md:62]:
```suggestion
- [Editor] You can now manage font properties in the editor.
```

File: i18n/fr/docusaurus-plugin-content-docs/version-22.10/releases/centreon-commercial-extensions.md [hunk 28]
```diff
@@ -17,6 +17,49 @@ Retrouvez plus de détails sur la version 22.10 dans notre [post de blog](https:
...
+- [Viewer] Fixed an issue that caused the unit not matching the axis on graph widget.
+
+#### Enhancements
+
+- [API] Improve api response when server is starting up.
+- [Editor] Manage font properties in editor.
+- [Editor] Manage view background color.
```
Comment (smau-centreon) [inline i18n/fr/docusaurus-plugin-content-docs/version-22.10/releases/centreon-commercial-extensions.md:60]:
```suggestion
- [Editor] You can now manage the view background color.
```

File: i18n/fr/docusaurus-plugin-content-docs/version-22.10/releases/centreon-commercial-extensions.md [hunk 29]
```diff
@@ -141,6 +201,12 @@ Release date: `October 26, 2022`
 
 ## Centreon Auto Discovery
 
+### 22.10.1
+
+Release date: `soon`
+
+- [UI] Fixed a cosmetic issue in Bam & Autodiscovery table header
```
Comment (smau-centreon) [inline i18n/fr/docusaurus-plugin-content-docs/version-22.10/releases/centreon-commercial-extensions.md:208]:
```suggestion
- [UI] Fixed a cosmetic issue in BAM and Autodiscovery table header.
```

File: versioned_docs/version-22.10/releases/centreon-commercial-extensions.md [hunk 30]
```diff
@@ -140,6 +200,12 @@ Release date: `October 26, 2022`
 
 ## Centreon Auto Discovery
 
+### 22.10.1
+
+Release date: `soon`
+
+- [UI] Fixed a cosmetic issue in Bam & Autodiscovery table header
```
Comment (smau-centreon) [inline versioned_docs/version-22.10/releases/centreon-commercial-extensions.md:207]:
```suggestion
- [UI] Fixed a cosmetic issue in BAM and Autodiscovery table header.
```

File: i18n/fr/docusaurus-plugin-content-docs/version-22.10/releases/centreon-commercial-extensions.md [hunk 31]
```diff
@@ -170,6 +236,14 @@ Release date: `October 26, 2022`
 
 ## Centreon Anomaly Detection
 
+### 22.10.2
+
+Release date: `soon`
+
+#### Bug fixes
+
+- [Install] Fixed an packaging issue that prevented anomaly detection from being installed.
```
Comment (smau-centreon) [inline i18n/fr/docusaurus-plugin-content-docs/version-22.10/releases/centreon-commercial-extensions.md:245]:
```suggestion
- [Install] Fixed a packaging issue that prevented Anomaly Detection from being installed.
```

**Review by smau-centreon — CHANGES_REQUESTED** [review]

File: i18n/fr/docusaurus-plugin-content-docs/version-22.10/releases/centreon-commercial-extensions.md (hunk 4 above)
Comment (tuntoja) [reply i18n/fr/docusaurus-plugin-content-docs/version-22.10/releases/centreon-commercial-extensions.md:31]: should it be this @nmagniezCentreon ?
Fixed the security issue CVE-2022-42889 (Text4shell).

File: i18n/fr/docusaurus-plugin-content-docs/version-22.10/releases/centreon-commercial-extensions.md (hunk 5 above)
Comment (tuntoja) [reply i18n/fr/docusaurus-plugin-content-docs/version-22.10/releases/centreon-commercial-extensions.md:28]: Will be removed

**Review by Jordan91 — DISMISSED** [review]

**Review by smau-centreon — APPROVED** [review]

**Review by nmagniezCentreon — APPROVED** [review]

File: i18n/fr/docusaurus-plugin-content-docs/version-22.10/releases/centreon-commercial-extensions.md [hunk 32]
```diff
@@ -57,13 +99,19 @@ The new MAP extension is now available in a full web version with a new server,
 
 ## Centreon MAP Legacy
 
+### 22.10.5
+
+Release date: `April 18, 2023`
```
Comment (tuntoja) [inline i18n/fr/docusaurus-plugin-content-docs/version-22.10/releases/centreon-commercial-extensions.md:104]:
```suggestion
Release date: `April 24, 2023`
```

File: i18n/fr/docusaurus-plugin-content-docs/version-22.10/releases/centreon-commercial-extensions.md [hunk 33]
```diff
@@ -100,6 +148,17 @@ Release date: `October 26, 2022`
 
 ## Centreon BAM
 
+### 22.10.2
+
+Release date: `April 18, 2023`
```
Comment (tuntoja) [inline i18n/fr/docusaurus-plugin-content-docs/version-22.10/releases/centreon-commercial-extensions.md:153]:
```suggestion
Release date: `April 24, 2023`
```

File: i18n/fr/docusaurus-plugin-content-docs/version-22.10/releases/centreon-commercial-extensions.md [hunk 34]
```diff
@@ -141,6 +200,12 @@ Release date: `October 26, 2022`
 
 ## Centreon Auto Discovery
 
+### 22.10.1
+
+Release date: `April 18, 2023`
```
Comment (tuntoja) [inline i18n/fr/docusaurus-plugin-content-docs/version-22.10/releases/centreon-commercial-extensions.md:205]:
```suggestion
Release date: `April 24, 2023`
```

File: i18n/fr/docusaurus-plugin-content-docs/version-22.10/releases/centreon-commercial-extensions.md [hunk 35]
```diff
@@ -170,6 +235,14 @@ Release date: `October 26, 2022`
 
 ## Centreon Anomaly Detection
 
+### 22.10.2
+
+Release date: `April 18, 2023`
```
Comment (tuntoja) [inline i18n/fr/docusaurus-plugin-content-docs/version-22.10/releases/centreon-commercial-extensions.md:240]:
```suggestion
Release date: `April 24, 2023`
```

File: versioned_docs/version-22.10/releases/centreon-commercial-extensions.md [hunk 36]
```diff
@@ -18,6 +18,48 @@ Read more about version 22.10 in our [blog post](https://www.centreon.com/en/blo
 
 ## Centreon MAP
 
+### 22.10.5
+
+Release date: `April 18, 2023`
```
Comment (tuntoja) [inline versioned_docs/version-22.10/releases/centreon-commercial-extensions.md:23]:
```suggestion
Release date: `April 24, 2023`
```

File: versioned_docs/version-22.10/releases/centreon-commercial-extensions.md [hunk 37]
```diff
@@ -58,15 +100,21 @@ The new MAP extension is now available in a full web version with a new server,
 
 ## Centreon MAP Legacy
 
+### 22.10.5
+
+Release date: `April 18, 2023`
```
Comment (tuntoja) [inline versioned_docs/version-22.10/releases/centreon-commercial-extensions.md:105]:
```suggestion
Release date: `April 24, 2023`
```

File: versioned_docs/version-22.10/releases/centreon-commercial-extensions.md [hunk 38]
```diff
@@ -101,6 +149,17 @@ Release date: `October 26, 2022`
 
 ## Centreon BAM
 
+### 22.10.2
+
+Release date: `April 18, 2023`
```
Comment (tuntoja) [inline versioned_docs/version-22.10/releases/centreon-commercial-extensions.md:154]:
```suggestion
Release date: `April 24, 2023`
```

File: versioned_docs/version-22.10/releases/centreon-commercial-extensions.md [hunk 39]
```diff
@@ -140,6 +201,12 @@ Release date: `October 26, 2022`
 
 ## Centreon Auto Discovery
 
+### 22.10.1
+
+Release date: `April 18, 2023`
```
Comment (tuntoja) [inline versioned_docs/version-22.10/releases/centreon-commercial-extensions.md:206]:
```suggestion
Release date: `April 24, 2023`
```

File: versioned_docs/version-22.10/releases/centreon-commercial-extensions.md [hunk 40]
```diff
@@ -169,6 +236,14 @@ Release date: `October 26, 2022`
 
 ## Centreon Anomaly Detection
 
+### 22.10.2
+
+Release date: `April 18, 2023`
```
Comment (tuntoja) [inline versioned_docs/version-22.10/releases/centreon-commercial-extensions.md:241]:
```suggestion
Release date: `April 24, 2023`
```

**Review by nmagniezCentreon — DISMISSED** [review]

File: i18n/fr/docusaurus-plugin-content-docs/version-22.10/releases/centreon-commercial-extensions.md [hunk 41]
```diff
@@ -17,6 +17,48 @@ Retrouvez plus de détails sur la version 22.10 dans notre [post de blog](https:
 
 ## Centreon MAP
 
+### 22.10.5
+
+Release date: `April 24, 2023`
+
+> You need to [execute the **configure.sh** script](../graph-views/map-web-install.md#étape-6--exécuter-le-script-configuresh) in order to apply some fixes provided by this release.
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs/version-22.10/releases/centreon-commercial-extensions.md:24]:
```suggestion
> You need to [execute the **configure.sh** script](../graph-views/map-web-install.md#step-6--execute-the-configuresh-script) in order to apply some fixes provided by this release.
```

**Review by nmagniezCentreon — DISMISSED** [review]

File: i18n/fr/docusaurus-plugin-content-docs/version-22.10/releases/centreon-commercial-extensions.md [hunk 42]
```diff
@@ -17,6 +17,48 @@ Retrouvez plus de détails sur la version 22.10 dans notre [post de blog](https:
 
 ## Centreon MAP
 
+### 22.10.5
+
+Release date: `April 24, 2023`
+
+> You need to [execute the **configure.sh** script](../graph-views/map-web-install.md#step-6--execute-the-configuresh-script) in order to apply some fixes provided by this release.
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs/version-22.10/releases/centreon-commercial-extensions.md:24]:
```suggestion
> You need to [execute the **configure.sh** script](../graph-views/map-web-install.md#étape-6--exécuter-le-script-configuresh) in order to apply some fixes provided by this release.
```

File: versioned_docs/version-22.10/releases/centreon-commercial-extensions.md [hunk 43]
```diff
@@ -18,6 +18,48 @@ Read more about version 22.10 in our [blog post](https://www.centreon.com/en/blo
 
 ## Centreon MAP
 
+### 22.10.5
+
+Release date: `April 24, 2023`
+
+> You need to [execute the **configure.sh** script](../graph-views/map-web-install.md#étape-6--exécuter-le-script-configuresh) in order to apply some fixes provided by this release.
```
Comment (cg-tw) [inline versioned_docs/version-22.10/releases/centreon-commercial-extensions.md:25]:
```suggestion
> You need to [execute the **configure.sh** script](../graph-views/map-web-install.md#step-6--execute-the-configuresh-script) in order to apply some fixes provided by this release.
```

**Pull request merged**